payroll-processor-functions/src/payroll-processor-functions/Features/Resources/Resource.cs

```diff
@@ -0,0 +1,17 @@
+namespace PayrollProcessor.Functions.Features.Resources
+{
+    public static class Resource
```

Comment:
This class helps reduced the scattered strings / stringly-typed values throughout the app.

Since the values are mostly used in attributes, we can't get clever with them and use something other than `const string`